syntax = "proto3";
package terragen;

message LoginRespondModel {
  bool success = 1;
  int32 udpPort = 2;
}